pkg/utils/dataset/lifecycle/node.go

```
@@ -120,67 +119,48 @@ func CanbeAssigned(runtimeInfo base.RuntimeInfoInterface, node v1.Node) bool {
-		labelExclusiveName = utils.GetExclusiveKey()
-	}
 
 	storageMap := tieredstore.GetLevelStorageMap(runtimeInfo)
 
-	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
-		nodeName := nodeToLabel.Name
-		node, err := kubeclient.GetNode(client, nodeName)
-		if err != nil {
-			log.Error(err, "GetNode In labelCacheNode")
-			return err
-		}
-		toUpdate := node.DeepCopy()
-		if toUpdate.Labels == nil {
-			toUpdate.Labels = make(map[string]string)
-		}
+	toUpdate := nodeToLabel.DeepCopy()
+	if toUpdate.Labels == nil {
+		toUpdate.Labels = make(map[string]string)
+	}
 
-		toUpdate.Labels[labelName] = "true"
-		toUpdate.Labels[labelCommonName] = "true"
-		if exclusiveness {
-			// toUpdate.Labels[labelExclusiveName] = fmt.Sprintf("%s_%s", runtimeInfo.GetNamespace(), runtimeInfo.GetName())
-			toUpdate.Labels[labelExclusiveName] = utils.GetExclusiveValue(runtimeInfo.GetNamespace(), runtimeInfo.GetName())
-		}
-		totalRequirement, err := resource.ParseQuantity("0Gi")
-		if err != nil {
-			log.Error(err, "Failed to parse the total requirement")
-		}
-		for key, requirement := range storageMap {
-			value := utils.TranformQuantityToUnits(requirement)
-			if key == common.MemoryCacheStore {
-				toUpdate.Labels[runtimeInfo.GetLabelnameForMemory()] = value
-			} else {
-				toUpdate.Labels[runtimeInfo.GetLabelnameForDisk()] = value
-			}
-			totalRequirement.Add(*requirement)
-		}
-		totalValue := utils.TranformQuantityToUnits(&totalRequirement)
-		toUpdate.Labels[runtimeInfo.GetLabelnameForTotal()] = totalValue
-
-		// toUpdate.Labels[labelNameToAdd] = "true"
-		err = client.Update(context.TODO(), toUpdate)
-		if err != nil {
-			log.Error(err, "LabelCachedNodes")
-			return err
+	toUpdate.Labels[labelName] = "true"
+	toUpdate.Labels[labelCommonName] = "true"
+	if exclusiveness {
+		toUpdate.Labels[utils.GetExclusiveKey()] = utils.GetExclusiveValue(runtimeInfo.GetNamespace(), runtimeInfo.GetName())
+	}
+
+	totalRequirement, err := resource.ParseQuantity("0Gi")
```

suggest 
```
totalRequirement := resource.MustParse("0Gi")
```
Because you know what you're doing, if there's a question instead of returning an Err but Panic

---

Good suggestion. Thx. Done.